docs/2.guide/2.directory-structure/3.app-config.md

```diff
@@ -99,7 +99,7 @@ Nuxt uses a custom merging strategy for the `AppConfig` within [the layers](/doc
 This strategy is implemented using a [Function Merger](https://github.com/unjs/defu#function-merger), which allows defining a custom merging strategy for every key in `app.config` that has an array as value.
 
 ::note
-The Function Merger should only be used in the base `app.config` of your application.
+The Function Merger should not be used in the default `app.config` of your application.
```

Comment:
It means it shall used in layers (not main project `app.vue`).

Maybe something like this?

```suggestion
The function merger can only be used in the extended layers and not the main `app.config` in project.
```

Reply:
Yes, sounds more clear 👍